src/main/java/payapal/Perfection.java

```diff
@@ -0,0 +1,42 @@
+package main.java.payapal;
+
+public class Perfection {
+
+    private static Perfection perf;
+
+    public synchronized static Perfection getPerf() {
+        if (perf == null) {
+            perf = new Perfection();
+        }
+        return perf;
+    }
+
+
+    public static boolean isPerfect(long candidate) {
```

Comment:
Please, correct me if I am wrong but I think there is no reason for this to be a class method. Class methods are those that you can invoke call without any instances. But, as per I can understand, if you need this feature, you will instance this class and then call the method. These kind of methods are called "instance methods" because you need an instance of the class before you can use them.